### PR TITLE
Add --detailed-exitcodes arg

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

**Comment by ares** [comment]
ACK (I should move code from bin script to hooks though since it's getting bigger and bigger)

**Comment by domcleal** [comment]
I guess that'd also fix http://projects.theforeman.org/issues/5488

File: bin/foreman-installer [hunk 1]
```diff
@@ -34,6 +34,10 @@ Kafo::KafoConfigure.app_option '--reset-foreman-db', :flag,
                                'manually. e.g. dropping can fail when DB is currently in use.',
     :default => false
 
+Kafo::KafoConfigure.app_option '--detailed-exitcodes', :flag,
+                               "Provide transaction information via exit codes, see puppet-agent(8)\n" +
+                               'full details.', :default => false
```
Comment (GregSutcliffe) [inline bin/foreman-installer]: nitpick, should read "for full details" ?

**Comment by ares** [comment]
@domcleal yes, you're right

File: bin/foreman-installer (hunk 1 above)
Comment (domcleal) [inline bin/foreman-installer]: @GregSutcliffe thanks, fixed

**Comment by GregSutcliffe** [comment]
Merged as 601047d, thanks!

**Pull request closed without merging**